src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -175,6 +175,14 @@
               </DataPlaneTraffic>
               <DataPlaneTraffic>
                 <template #actions>
+                  <KInputSwitch
+                    v-model="showInactive"
```

Comment:
If https://github.com/kumahq/kuma-gui/pull/1966 goes in first, this v-model will be switched ot just use `route.params.inactive`, which following #1966 has a two way binding. If #1966 goes in last, there'll be another PR to change this up.